src/mmrelay/cli.py

```
@@ -1752,67 +1729,61 @@ def handle_config_diagnose(args: argparse.Namespace) -> int:
-
-        success = install_service()
-        import sys
+    if args_dict.get("install_service"):
+        logger.warning(get_deprecation_warning("--install-service"))
+        try:
+            from mmrelay.setup_utils import install_service
 
-        sys.exit(0 if success else 1)
+            return 0 if install_service() else 1
+        except ImportError as e:
+            logger.error(f"Error importing setup utilities: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)